tests/data/granule.umm_json

```diff
@@ -132,5 +132,24 @@
             "Date": "2024-03-02T22:12:36.000Z",
             "Type": "Update"
         }
-    ]
+    ],
+    "Platforms": [
+            "Sentinel-1A",
+            "Sentinel-1A"
+            ],
+    "OrbitDirectionTypeEnum": "D",
+    "InputGranules": ["S1A_IW_SLC__1SDV_20240212T032621_20240212T032649_052520_065A36_7976",
+                      "S1A_IW_SLC__1SDV_20240212T032646_20240212T032714_052520_065A36_4F51",
+                      "S1A_IW_SLC__1SDV_20240107T032622_20240107T032650_051995_06487A_6B83",
+                      "S1A_IW_SLC__1SDV_20240107T032648_20240107T032715_051995_06487A_DBAE"
+                      ],
+    "AdditionalAttributes": [
+            {"Name": "ASCENDING_DESCENDING", "Values": ["descending"]},
+            {"Name": "BEAM_MODE",  "Values": ["IW"]},
+            {"Name": "POLARIZATION", "Values": ["VV"]},
+            {"Name": "PERPENDICULAR_BASELINE", "Values": [56.0596]},
+            {"Name": "VERSION", "Values": ["3.0.0"]},
+            {"Name": "FRAME_NUMBER", "Values": [19116]},
+            {"Name": "TEMPORAL_BASELINE_DAYS", "Values": [36]}
```

Comment:
Per the umm-g schema, we need to report all attributes values as strings: https://git.earthdata.nasa.gov/projects/EMFD/repos/unified-metadata-model/browse/granule/v1.6.5/umm-g-json-schema.json#1057

```suggestion
            {"Name": "PERPENDICULAR_BASELINE", "Values": ["56.0596"]},
            {"Name": "VERSION", "Values": ["3.0.0"]},
            {"Name": "FRAME_NUMBER", "Values": ["19116"]},
            {"Name": "TEMPORAL_BASELINE_DAYS", "Values": ["36"]}
```